src/platyPS/platyPS.psm1

```diff
@@ -570,6 +570,9 @@ function New-ExternalHelp
 
         [System.Text.Encoding]$Encoding = $script:UTF8_NO_BOM,
 
+        [Parameter(HelpMessage="Specifies the encoding for get raw content, with a default value of UTF8")]
+        [Microsoft.PowerShell.Commands.FileSystemCmdletProviderEncoding]$Encode = [Microsoft.PowerShell.Commands.FileSystemCmdletProviderEncoding]::UTF8,
```

Comment:
We are using `[System.Text.Encoding]` for encoding, because `FileSystemCmdletProviderEncoding ` doesn't support no-BOM